src/shared-components/container/style.ts

```diff
@@ -65,20 +64,11 @@ const Image = styled.img`
   object-fit: cover;
 `;
 
-/**
- * This is the type returned by `styled.div`
- *
- * @see `node_modules/@emotion/styled-base/types/index.d.ts`
- */
-type StyledContainer = StyledComponent<
-  DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
-  {
-    type?: ContainerType;
-  },
-  { theme?: ThemeType }
->;
-
-type CompositeContainer = StyledContainer & {
+const ContainerComponentForTypesOnly = styled.div<{
```

Comment:
Rather than import the "exotic" types used, we just set up a variable to use the type definition. I was not able to use the value directly without first setting it in a variable.